src/js/components/Form/stories/ValidateOnBlur.js

```diff
@@ -37,12 +37,12 @@ export const ValidateOnBlur = () => (
               return undefined;
             },
           ]}
-        />
-
-        <FormField label="Email" name="email" required>
-          <TextInput name="email" aria-label="email" type="email" />
+        >
+          <TextInput aria-label="name" id="name" aria-required name="name" />
```

Comment:
Shouldn't need `aria-label` given the `htmlFor`

Reply:
done